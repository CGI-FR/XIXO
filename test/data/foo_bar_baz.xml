<root>
  <foo>
    <bar>a</bar>
    <baz fuz="faz">z</baz>
  </foo>
  <foo>
    <bar>b</bar>
  </foo>
  <baz>
    <bar>c</bar>
  </baz>
</root>
