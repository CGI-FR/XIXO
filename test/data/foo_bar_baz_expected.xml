<root>
  <foo>
  <bar>A</bar>
  <baz fuz="faz">z</baz>
</foo>
  <foo>
  <bar>B</bar>
</foo>
  <baz>
    <bar>c</bar>
  </baz>
</root>
